lib/code_corps_web/controllers/password_reset_controller.ex

```diff
@@ -29,6 +30,7 @@ defmodule CodeCorpsWeb.PasswordResetController do
       |> render("show.json", token: auth_token, user_id: updated_user.id, email: updated_user.email)
     else
       {:error, %Changeset{} = changeset} -> conn |> put_status(422) |> render(CodeCorpsWeb.ErrorView, :errors, data: changeset)
+      {:error, _} -> conn |> put_status(:not_found) |> render(CodeCorpsWeb.ErrorView, "404.json")
```

Comment:
Having trouble testing this.  Only works if token was expired or perhaps the `secret_key_base` was different.

Reply:
Could you change the `env` to change the `secret_key_base` and reset it? Or manually set the token to be expired?

Reply:
Yeah that is a good idea.  I guess I just don't know how to `put` this environmental variable.
```
    secret_key = Application.get_env(:guardian, Guardian)[:secret_key]
    Application.put_env(:guardian, :secret_key, "1234")
```

Reply:
Anyways, I configured the password_reset_timeout in the config file and that work!  Thanks for the help!